Faokryn.yaml: Added teamprop1, fixed meetup1 & bugfix

faokryn opened the pull request.

faokryn committed:
Faokryn.yaml: Added teamprop1, fixed meetup1 & bugfix

faokryn committed:
Added commarch and commarch slides to my yaml (Faokryn.yaml)

decause commented:
<code>commarch</code> should be <code>commarchreport</code>

Fix and I'll merge :wrench:

faokryn committed:
Merge branch 'master' of https://github.com/decause/hflossk

faokryn committed:
Fixed "commarch" to "commarchreport" in Faokryn.yaml

faokryn committed:
Merge branch 'master' of https://github.com/decause/hflossk

faokryn committed:
Modified Faokryn.yaml

Added meetup2, hackathon2, and hackathon3
Changed references to faokryn-dev.tumblr.com to faokryn.com (going to faokryn-dev.tumblr.com
will still work, as it will redirect to the same page on faokryn.com)

decause commented:
:bird:
